src/components/StoryList.component.js

```diff
@@ -115,6 +130,9 @@ function SidebarContent({
             {story.name}
           </div>
           <div className="divider" />
+
+          <div onClick={() => onClickEdit(story.id)}>Edit</div>
```

Comment:
These should go before the divider - it's confusing which edit/deletes refer to which story